corehq/apps/geospatial/templates/geospatial/partials/review_assignment_modal.html

```diff
@@ -37,8 +37,10 @@ <h4 class="modal-title">
                 <table class="table table-striped table-bordered">
                     <thead>
                     <tr>
-                        <th>
-                            <input type="checkbox" data-bind="checked: isAllChecked" />
+                        <th class="col-sm-3">
+                            <span>{% trans 'Select' %}</span>
+                            <button class="btn btn-xs btn-default" data-bind="click: toggleSelectAll(true)">{% trans 'All' %}</button>
```

Comment:
I like the idea of being explicit , however feel like it might not be needed in this case.
Having a select checkbox in header is a  common UX pattern .
Does not impact much, just reduces a couple of buttons. I am fine with the current way also.

Reply:
The pattern of having an all/none button is something that gets used in other places in HQ, such as the copy cases feature. Using a select checkbox does make the UI more concise, however I'm in favour of rather keeping the UI more explicit given that this isn't a pattern we very much use in HQ.